### DISCUSSION TITLE
Time ago in words

### DISCUSSION THREAD

**Discussion by gnoshme** [magicbug/Cloudlog#1983] [Ideas / Feature Requests]
I'd really like a time_ago_in_words output along with QSO Date/Time in most places.. e.g. small font (about 3 hours ago).   This is especially useful in the result of the callsign lookup on the QSO dialogs for people way outside UTC. My brain doesn't instantly make sense of the Date/Time timestamp.   Added this in via the HTML generated by controllers/Logbook.php for that specific spot and for me it's a good feature.

Keifer / K3IFR

**Comment by phl0** [comment]
Like that?

![Screenshot from 2023-02-05 19-55-32](https://user-images.githubusercontent.com/7112907/216839111-c7a989fd-b1b5-4633-a67c-984c3d2a38cf.png)

**Reply by gnoshme** [reply]
More like this.  If that's on a hover it's not as useful for someone who is a keyboard warrior and using footswitch for the radio.

![image](https://user-images.githubusercontent.com/2192/216840483-654e95a1-dbed-403b-8c9b-6d745cf78cbb.png)

**Reply by gnoshme** [reply]
.. smaller is better

![image](https://user-images.githubusercontent.com/2192/216842076-05145748-2366-44f0-aa65-000dcfe5b332.png)

**Comment by magicbug** [comment]
I'm trying to understand why you would want this next to the date/time that already exists.

**Reply by gnoshme** [reply]
When you're not close to UTC the date/time exists right there sure, but it's not instantly obvious what that date time is relative to now.  Contacts days apart it's obvious.  Contacts hours apart it isn't.  When I'm hunting POTA here it's not unusual to come across the same person doing multiple parks or the same park a day apart.  It's nice to have an instant time ago in words to mentally untangle that.  On top of that, since rails made time_ago_in_words a native function I've got used to implementing it wherever it is even slightly useful so I just like it.  I've implemented it everywhere in Cloudlog and I see value in it.
